src/lib.rs

```diff
@@ -177,6 +294,14 @@ impl InliningResult {
         !self.errors.is_empty()
     }
 
+    /// Whether paths were annotated using `InlinerBuilder::annotate_paths`.
+    ///
+    /// If this is `true`, `find_mod_path` may be used to figure out the path a module is defined
+    /// in.
+    pub fn paths_annotated(&self) -> bool {
```

Comment:
How would you feel about this being called `has_path_annotations`? In my opinion that makes it clearer this is describing the current state of something.